docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -13,26 +31,43 @@ Every example has a `docker-compose.yaml` manifest that includes all of the opti
 
 The Tempo examples running with Docker using docker-compose include a version of Tempo and a storage configuration suitable for testing or development.
 
-This quick start guide uses the `local` example running Tempo as a single binary (monolithic). Any data is stored locally in the `tempo-data` folder.
-To learn more, read the [local storage example README](https://github.com/grafana/tempo/blob/main/example/docker-compose/local).
 
-## Alternative: Complete MLTP example
-
-If you would like to use a demo with Tempo, Mimir, Loki, and Grafana, then try the [Introduction to Metrics, Logs, Traces, and Profiling in Grafana](https://github.com/grafana/intro-to-mlt).
-`Intro-to-mltp` provides a self-contained environment for learning about Mimir, Loki, Tempo, and Grafana.
-The project includes detailed explanations of each component and annotated configurations for a single-instance deployment.
-Data from `intro-to-mltp` can also be pushed to Grafana Cloud.
+<!-- INTERACTIVE ignore START -->
 
 ## Before you begin
 
 To follow this guide, you need:
 
 - Git
-- Docker
-- The docker-compose plugin
+- [Docker](https://docs.docker.com/compose/install/)
+- The Docker Compose plugin (included with Docker Desktop)
+
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Quick start for Tempo](https://killercoda.com/grafana-labs/course/tempo/quick-start).
+
+It's a fully configured environment with all the dependencies installed.
+
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
```

Comment:
```suggestion

Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
```

Extra CR/LF

Alloy includes a banner image at this point. There's no banner here?

Also, this is new compared to the Alloy version. Should I include this over in the Alloy tutorials?

Reply:
@Jayclifford345 What do you think?

Reply:
Yeah, more than happy to have the banner removed and this section updated. I still think some form of icon or branding, later on, to differentiate it from a normal admonition makes users know this tutorial includes a sandbox from the initial glance.